sys/pm_layered/pm.c

```diff
@@ -100,7 +102,7 @@ pm_blocker_t pm_get_blocker(void)
     pm_blocker_t result;
 
     unsigned state = irq_disable();
-    memcpy(&result, &pm_blocker, sizeof(result));
+    result = pm_blocker;
```

Comment:
You can remove `<string.h>` include, then.

Maybe also put this in its own commit? Or is this change actually related to alignment?

Reply:
i will remove the <string.h>
btw in there was a `memcpy` in the assembly of the samr0 before this PR but the rewrite of the line is not connected to that no longer beeing part of the assembly.

Reply:
these are now 3 commits that separate the changes.